.github/dependabot.yml

```diff
@@ -12,3 +7,30 @@ updates:
       interval: "daily"
     open-pull-requests-limit: 5
     target-branch: "master"
+    groups:
+      jakarta:
+        patterns:
+        - "jakarta.*"
+        update-types:
+        - "patch"
+      microprofile:
+        patterns:
+        - "org.eclipse.microprofile*"
+        update-types:
+        - "patch"
+    ignore:
+      - dependency-name: "jakarta.*"
+        update-types: ["version-update:semver-major"]
+      - dependency-name: "jakarta.*"
+        update-types: ["version-update:semver-minor"]
+      - dependency-name: org.eclipse.microprofile*"
+        update-types: ["version-update:semver-major"]
+      - dependency-name: "org.eclipse.microprofile*"
+        update-types: ["version-update:semver-minor"]
+  - package-ecosystem: "gitsubmodule"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Do we need to define a schedule? It seems a bit odd to scan daily to ignore everything